docs/05-rfcs/2023-01-21-wing-plugins.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Wing Plugins
+description: How to implement wing plugins
+---
+
+# Wing Plugin System
```

Comment:
I am not super comfortable with the name "plugin", but I am okay with it for now. Just wanted to share that :-)

Reply:
I think I started less comfortable with it, but the more research I have done the more comfortable I have become with it. But yea I totally get it, If we want to get to a point where we have a really nice declarative syntax like you described here: https://github.com/winglang/wing/issues/493 then the name plugin does not work in my opinion. However, this writing raw JS files fits the plugin theme IMO but It may be opening too big of a can of worms (security wise).